### PR TITLE
BRO Queen Kayla bin-Kroog

### PR DESCRIPTION
**Pull request by Simisays**
Took me a while to figure this out

### CODE REVIEW

**Review by tool4ever — DISMISSED** [review]
Looks good, nice work

**Comment by Hanmac** [comment]
we probably need new rules for this, but i think it should be allowed to get cards that got exiled from an replacement effect

File: forge-gui/res/cardsfolder/upcoming/queen_kayla_bin-kroog.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+SVar:DBDraw:DB$ Draw | Defined$ You | NumCards$ X | SubAbility$ DBChooseCardOne
+SVar:DBChooseCardOne:DB$ ChooseCard | Defined$ You | Choices$ Artifact.IsRemembered+cmcEQ1,Creature.IsRemembered+cmcEQ1 | ImprintChosen$ True | ChoiceTitle$ You may choose an artifact or creature card with mana value 1 you discarded this way | ChoiceZone$ Graveyard | AILogic$ BestCard | SubAbility$ DBChooseCardTwo 
+SVar:DBChooseCardTwo:DB$ ChooseCard | Defined$ You | Choices$ Artifact.IsRemembered+cmcEQ2,Creature.IsRemembered+cmcEQ2 | ImprintChosen$ True  | ChoiceTitle$ You may choose an artifact or creature card with mana value 2 you discarded this way | ChoiceZone$ Graveyard | AILogic$ BestCard | SubAbility$ DBChooseCardThree
+SVar:DBChooseCardThree:DB$ ChooseCard | Defined$ You | Choices$ Artifact.IsRemembered+cmcEQ3,Creature.IsRemembered+cmcEQ3 | ImprintChosen$ True | ChoiceTitle$ You may choose an artifact or creature card with mana value 3 you discarded this way | ChoiceZone$ Graveyard | AILogic$ BestCard | SubAbility$ DBChangeZoneAll
+SVar:DBChangeZoneAll:DB$ ChangeZoneAll | Origin$ Graveyard | ChangeType$ Card.IsImprinted | Destination$ Battlefield | SubAbility$ DBCleanup
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/queen_kayla_bin-kroog.txt:10]: @tool4ever @Hanmac is it more efficient to do it this way or with `DB$ ChangeZone | Defined$ Imprinted` ?

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/queen_kayla_bin-kroog.txt:10]: PS nice work @Simisays 💪

Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/queen_kayla_bin-kroog.txt:10]: `Defined$ Imprinted` is cleaner. Assuming the cards are in a public zone (graveyard or exile?)

**Review by Northmoc — APPROVED** [review]
Looks good! 👍

**Pull request merged**